geo/src/main/java/org/odk/collect/geo/selection/SelectionMapFragment.kt

```diff
@@ -354,6 +350,9 @@ class SelectionMapFragment(
             itemsByFeatureId[featureId] = item
             points.add(point)
         }
+        selectedFeatureViewModel.getSelectedFeatureId()?.let {
```

Comment:
I'm confused about why this is needed at all. I would expect https://github.com/getodk/collect/pull/5095/commits/14f6a6a218ad4ea8f9f8d8e584197b577e2c3339#diff-41e26ac0b5e19db6790e339c006bc7c1ce451d326733985f3309aa5f3f3d6d89R317 to do the actual feature click. And in fact, I would expect that `getSelectedFeatureId` is always null here because it gets set in `onFeatureClicked`.

Reply:
It is here to keep already selected point highlighted (big marker and the bottom sheet) after recreating - rotating a device for example.